cfpb/capital-framework
# Style headings within table header cells only as standard table headers

We have a page in which we'd like to use `h5` elements inside of a `th`. See https://GHE/CFGOV/platform/issues/2703 for background.

The issue is that, currently, putting a heading in a `th` messes with the font size and weight and brings in unwanted margin. This PR addresses that.

## Additions

- Adds rules to override standard heading styling for headings within `thead th` elements

## Testing

1. Pull branch
1. `gulp`
1. `cd docs && bundle exec jekyll serve`
1. Visit http://localhost:4000/components/cf-tables/
1. Scroll down to a table, inspect a header cell, edit its HTML, and change it so that its text is enclosed in a heading element
1. See that the styling does not change

## Checklist

- [x] PR has been tagged with the type of change it includes
      (breaking, enhancement, bug, documentation or internal)
- [x] Project documentation has been updated in this PR with the `gulp docs` command

## Testing checklist

### Browsers


### Accessibility


### Other
